ext/pcre/config0.m4

```diff
@@ -83,31 +83,15 @@ else
 fi
 
 PHP_ARG_WITH([pcre-valgrind],,
-  [AS_HELP_STRING([--with-pcre-valgrind=DIR],
+  [AS_HELP_STRING([--with-pcre-valgrind],
```

Comment:
We might want to piggy-back off the `--with-valgrind` option here. Historically `--with-pcre-valgrind` came first, but now that we have `--with-valgrind` I think it makes sense to make it control both places.

Reply:
Yes, agreed.